website/docs/r/virtual_machine_restore_point.html.markdown

```diff
@@ -95,9 +95,9 @@ The following arguments are supported:
 
 * `name` - (Required) Specifies the name of the Virtual Machine Restore Point. Changing this forces a new resource to be created.
 
-* `restore_point_collection_id` - (Required) Specifies the name of the Virtual Machine Restore Point Collection the Virtual Machine Restore Point will be associated with. Changing this forces a new resource to be created.
+* `virtual_machine_restore_point_collection_id` - (Required) Specifies the ID of the Virtual Machine Restore Point Collection the Virtual Machine Restore Point will be associated with. Changing this forces a new resource to be created.
 
-* `crash_consistency_mode_enabled` - (Optional) Is Crash Consistent the Consistency Mode of the Virtual Machine Restore Point. Defaults to `false`. Changing this forces a new resource to be created.
+* `crash_consistency_mode_enabled` - (Optional) Whether the Consistency Mode of the Virtual Machine Restore Point is set to `CrashConsistent`. Defaults to `false`. Changing this forces a new resource to be created.
```

Comment:
I modified this line that isn't directly related to the PR just to make the wording slightly clearer.